packages/grafana-ui/package.json

```diff
@@ -75,6 +75,10 @@
     "@hello-pangea/dnd": "18.0.1",
     "@monaco-editor/react": "4.7.0",
     "@popperjs/core": "2.11.8",
+    "@rc-component/drawer": "1.3.0",
+    "@rc-component/picker": "1.7.1",
+    "@rc-component/slider": "1.0.1",
+    "@rc-component/tooltip": "1.4.0",
```

Comment:
it's bothering me that they didn't keep the version number. rc-drawer v7 -> @rc-component/drawer v1???